docs/testing/testplan.md

```diff
@@ -0,0 +1,427 @@
+# Web Monetization Extension Test Plan
+
+## Introduction
+
+The purpose of this document is to guide end-user testing of the Web Monetization browser extension.
+We focus on functional test cases. The goal is to report bugs, issues or differences observed from the expected behavior.
+
+## Audience
+
+Any person interested in testing the beta release of the Web Monetization extension.
+
+## Objectives
+
+1. Validating that, once installed, users can configure Web Monetization settings on the extension.
+2. Validating that once the extension is successfully connected to a digital wallet, a user is able to configure payments or disconnect from the wallet.
+3. When the extension facilitates paying a monetized web page, check how payments get distributed between the receiving wallets on the page.
+
+## Dependencies
+
+### Digital Wallets
+
+The main prerequisite for sending or receiving Web Monetization payments is to have a Web Monetization-enabled digital wallet. Web Monetization-enabled digital wallets are provided by licensed service providers. The wallet providers are regulated by the laws of the countries in which they operate.
+
+Below is a list of the available wallet providers if you want to use real money:
+
+- [Fynbos](https://wallet.fynbos.app)
+- [GateHub](https://gatehub.net)
+
+Wallet availability in a specific country, or availability in particular currency depends on the wallet provider.  
+We maintain a list of [Web Monetization compatible wallets](https://webmonetization.org/docs/resources/op-wallets/).
+
+#### Test Digital Wallet
+
+You have the option to use “play” money from the [Interledger Test Wallet application](https://rafiki.money/).  
+Here you can set up an account that is enabled for Web Monetization, and other Interledger functionality, without involving real money.
+
+## Testing
+
+### Environment and Software
+
+The resources and tools that you will need to test Web Monetization depend on your desired role, and what you want to test.
```

Comment:
I think we can remove references to desired roles since there's only one role covered in this test plan. I suggest removing the entire sentence on line 41, then editing line 42 to something like "This test plan covers scenarios where you, as a website visitor, want to pay web monetized sites."

Reply:
Done